src/challenge.ts

```diff
@@ -4,7 +4,7 @@ import { queryString } from "./queryString";
 import { AxiosOptions, Chain, PayoutType, Environment, ChallengeState } from "./types";
 import { AxiosResponse } from "axios";
 
-interface ChallengeResponse {
+export type ChallengeResponse = {
```

Comment:
updated the code using `type` instead of `interface` for 2 reasons: 
1. to align with other exported types
2. in VSCode, type alias will display more details of the type when hover over the type

`interface` when hover in VSCode
<img width="614" alt="image" src="https://github.com/Ludex-Labs/ludex-sdk-js/assets/105812701/b67db650-a330-45c3-bcb7-58afaa0dc1aa">

`type` when hover in VSCode
<img width="573" alt="image" src="https://github.com/Ludex-Labs/ludex-sdk-js/assets/105812701/e1c30c47-a6fb-4fdd-bb68-3d18a30c5834">